wordpress-mobile/WordPress-iOS
# WordPress Share Extension: Swift 4.0 Support

### Details:
This PR includes #8161 and #8159. Please, review + merge afterwards!

- Innocuous changes: just incrementing the lastSwiftMigration Flag
- Fixed a warning in our CocoaLumberjack Wrapper (contained by the main project, but used all over!)

Needs Review: @diegoreymendez 
Thanks!!


### To test:
1. Verify that the app still builds correctly
2. Switch the Swift Version to Mark 4.0 in these targets: **[WordPressShared, WordPressKit, WordPressShareExtension]**
3. Verify that the app still builds fine!
